src/data_provider/src/sysInfoLinux.cpp

```diff
@@ -400,6 +399,109 @@ nlohmann::json SysInfo::getNetworks() const
     return networks;
 }
 
+std::string parseProcessInfo(const std::filesystem::path path)
+{
+    // Get stat file content.
+    std::string processInfo { UNKNOWN_VALUE };
+    const std::string statContent {Utils::getFileContent(path)};
+
+    const size_t openParenthesisPos {statContent.find("(")};
+    const size_t closeParenthesisPos {statContent.find(")")};
+
+    if (openParenthesisPos != std::string::npos && closeParenthesisPos != std::string::npos)
+    {
+        processInfo = statContent.substr(openParenthesisPos + 1, closeParenthesisPos - openParenthesisPos - 1);
+    }
+
+    return processInfo;
+}
+
+void findInodeMatch(const std::filesystem::path path, std::vector<int64_t>& inodes, bool& matchInode, const nlohmann::json& portsInfo)
```

Comment:
```suggestion
void findInodeMatch(const std::filesystem::path& path, std::vector<int64_t>& inodes, bool& matchInode, const nlohmann::json& portsInfo)
```

Reply:
avoid copy, size of of std::fs::path is 40bytes, and can cause memory fragementation.